### ISSUE TITLE
Error when generating ParameterDatabase.sopd

### ISSUE THREAD

**Issue by mariomadproductions** [jupahe64/Spotlight#10]
Spotlight Version: v1.0
OS: Windows 10 (vmware virtual machine) or Windows 8.1 (real machine)
.NET Framework: v4.8

When I select "yes" to generating ParameterDatabase.sopd, it gives this error:
[error.txt](https://github.com/jupahe64/Spotlight/files/6086025/error.txt)

After this, the loading bar never stops, and a blank window appears.

Even when I put a pre-made one in the folder, it still tries to generate it.

**Comment by hypershadow** [comment]
I too have this problem. Was it ever solved?

**Comment by Lord-Giganticus** [comment]
It was not, and will likely not ever be fixed.

**Comment by mariomadproductions** [comment]
> It was not, and will likely not ever be fixed.
Why's that?

**Comment by mariomadproductions** [comment]
> It was not, and will likely not ever be fixed.

Why's that?

**Comment by Lord-Giganticus** [comment]
TLDR: the repo as it stands is basically abandoned. It's only been receiving random updates. If I'm going to be honest, you have a better chance using a pre-1.0.0.0 version of spotlight as it doesn't have as many still unfixed bugs

**Comment by mariomadproductions** [comment]
Ah I see.

**Comment by SuperHackio** [comment]
I tried this just now and I didn't get any errors.

Make of that what you will.
